pkg/openstack/common.go

```diff
@@ -290,6 +291,22 @@ func EnsureEndpointConfig(
 					} else if (ctrlResult != ctrl.Result{}) {
 						return endpoints, ctrlResult, nil
 					}
+					// Delete the issued certificate if it exists
+					certName := fmt.Sprintf("%s-route", ed.Name)
+					cert := certmanager.NewCertificate(
+						&certmgrv1.Certificate{
+							ObjectMeta: metav1.ObjectMeta{
+								Name:      certName,
+								Namespace: ed.Namespace,
+							},
+						},
+						5*time.Second,
+					)
+					err = cert.Delete(ctx, helper)
+					if err != nil {
+						helper.GetLogger().Error(err, fmt.Sprintf("Failed to delete unused route certificate %s", certName))
```

Comment:
its almost the same as the error message returned by the cert.Delete(), do we need this additional message? https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/certmanager/certificate.go#L160